docs/api-docs/payments/payments-api-overview.mdx

```diff
@@ -113,9 +113,14 @@ The payment gateway must support making charges to **stored payment instruments*
 The store must:
 * be on a BigCommerce Plus plan or higher;
 * use Optimized One-Page Checkout;
-* have stored credit cards enabled.
+* have stored credit cards and/or stored PayPal accounts enabled.
+
+**Enabling Stored Credit Cards**\
+Use the store's control panel to enable charging stored credit cards. Navigate to **Settings > Setup > Payments** and click the tab for your payment gateway. Toggle the switch to enable Stored Credit Cards and click **Save**.
+
+**Enabling Stored PayPal Accounts**\
```

Comment:
```suggestion
**Enabling Stored PayPal Accounts**
```

Reply:
I added the back slash to start a new line. Here is the preview.
![Screenshot 2024-02-05 at 1 55 05 PM](https://github.com/bigcommerce/docs/assets/64287776/a8ad7fe3-213d-4f99-86d2-32524a7f06c4)

Reply:
v good, i know you always check the deploys so we can fix if it's different.